src/olm/session/mod.rs

```diff
@@ -327,8 +327,8 @@ impl Session {
         #[derive(Debug, Zeroize)]
         #[zeroize(drop)]
         struct ReceivingChain {
```

Comment:
Didn't you intend to rename those so it's clearer they are libolm specific?

Reply:
I think there may be more renames necessary for clarity so I decided not to bother with it in this PR and leave it for another one.